app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,18 @@
+class SessionsController < ApplicationController
+  def create
+    body = {
+      client_id: ENV['GOOGLE_CLIENT_ID'],
+      client_secret: ENV['GOOGLE_CLIENT_SECRET'],
```

Comment:
I think Phil mentioned he would take these out as they are unnecessary in our morning oauth session